src/js/components/CheckBox/stories/InsideFormField.js

```diff
@@ -32,6 +32,23 @@ export const InsideFormField = props => (
             />
           </Box>
         </FormField>
+        <FormField
+          label="Toggle fill"
+          name="toggle"
+          htmlFor="check-box-toggle"
+          {...props}
+        >
+          <Box pad={{ horizontal: 'small', vertical: 'xsmall' }}>
+            <CheckBox
+              id="check-box-toggle"
+              name="toggle"
```

Comment:
Seems like you need to update the id/name stuff here because clicking on this toggle is just toggling the first one